packages/editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@portabletext/editor",
-  "version": "1.1.11",
+  "version": "1.1.12-canary.1",
```

Comment:
Why canary? What complications will this bring to the ordinary release process?

Reply:
No complications, release please ignores the version used here and follows the one in the manifest.
I kept it in the version field for transparency on the fact that I published a canary of this PR 😌